inf-clojure.el

```diff
@@ -833,12 +839,12 @@ process buffer for a list of commands.)"
             (cmdlist (if (consp cmd)
                          (list cmd)
                        (split-string-and-unquote cmd)))
-            (repl-type (or inf-clojure-socket-repl-type
-                           (unless prefix-arg
+            (repl-type (or (unless prefix-arg
                              inf-clojure-custom-repl-type)
-                        (car (rassoc cmd inf-clojure-startup-forms))
-                        (inf-clojure--prompt-repl-type))))
-        (message "Starting Clojure REPL via `%s'..." cmd)
+                           (car (rassoc cmd inf-clojure-startup-forms))
+                           (inf-clojure--prompt-repl-type))))
+        (unless inf-clojure--suppress-connect-message-p
```

Comment:
I'm not a big fan of dynamic vars and I'd probably remove this altogether and replace it with some optional param for the containing function.

Reply:
updated to have an optional passed arg.